### ISSUE TITLE
When using Library Browser, I get error:  'can't call method "ssl_opts" ... '

### ISSUE THREAD

**Issue by homerhanumat** [openwebwork/webwork2#714]
My issue may be related to [this one](https://github.com/openwebwork/webwork2/issues/567).

At my College we just installed WeBWorK 2.12 on an Ubuntu 14.04 server.  (SSL has not yet been hooked up so we access via http:// ...).  Whenever we pick a choice from any menu in the Library Browser (e.g., `Add Problems to Target Set` or `Subject`), we get the following error:

```
Error messages

    Can't call method "ssl_opts" on an undefined value at /usr/share/perl5/SOAP/Lite.pm line 482. 

Call stack

The information below can help locate the source of the problem.

    in SOAP::Transport::__ANON__ called at line 280 of /opt/webwork/webwork2/lib/WebworkClient.pm
    in WebworkClient::xmlrpcCall called at line 336 of /opt/webwork/webwork2/lib/WeBWorK/ContentGenerator/instructorXMLHandler.pm
    in WeBWorK::ContentGenerator::instructorXMLHandler::pre_header_initialize called at line 214 of /opt/webwork/webwork2/lib/WeBWorK/ContentGenerator.pm
    in WeBWorK::ContentGenerator::go called at line 382 of /opt/webwork/webwork2/lib/WeBWorK.pm
```

In the browser (using developer tools) i see  in the expected error message:

```
SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data
```

The `apache2/error.log` file gives the same info but with an additional remark at the beginning about an uninitialized value:

```
Use of uninitialized value in subroutine entry at /usr/share/perl5/HTML/Scrubber.pm line 181.
[Wed Jun 22 07:49:15.439436 2016] [perl:error] [pid 1712] [client 10.240.1.10:56367] [/webwork2/instructorXMLHandler] Can't call method "ssl_opts" on an undefined value at /usr/share/perl5/SOAP/Lite.pm line 482.\n * in SOAP::Transport::__ANON__ called at line 280 of /opt/webwork/webwork2/lib/WebworkClient.pm\n * in WebworkClient::xmlrpcCall called at line 336 of /opt/webwork/webwork2/lib/WeBWorK/ContentGenerator/instructorXMLHandler.pm\n * in WeBWorK::ContentGenerator::instructorXMLHandler::pre_header_initialize called at line 214 of /opt/webwork/webwork2/lib/WeBWorK/ContentGenerator.pm\n * in WeBWorK::ContentGenerator::go called at line 382 of /opt/webwork/webwork2/lib/WeBWorK.pm, referer: http://webwork.georgetowncollege.edu/webwork2/test-homer/instructor/setmaker/?effectiveUser=hwhite0&key=NayYGbZCydiEYTxzpphmGyVOgtEjOSpC&user=hwhite0
```

Thanks,
Homer White

**Comment by goehle** [comment]
I would run the "check_modules.pl" script to see if you have any missing or outdated modules and would double check for typos in the server address in your site.conf file.  These kinds of errors are usually caused by either missing or outdated perl packages or misconfigurations.

**Comment by homerhanumat** [comment]
The URL in site.conf is correct.  As for Perl modules, many were out of date as the OS version is 14.04 rather than 16.04.  When I updated modules I got a couple of failures, including a dependency of Soap::Lite, which is implicated in the error message.  I force-installed this dependency, but the error persists.

The system is not in production yet:  would it be best simply to start from scratch with Ubuntu 16.04?

Homer
